### ISSUE TITLE
Certificate of Eligibility | Content & IA review from Benefits Team 1

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#43350]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more new unauthenticated (static) VA.gov pages
- [x] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [ ] Launching a new online form, app, or tool on VA.gov
- [ ] Making revisions to an existing online form, app, or tool on VA.gov
- [ ] Something else (please add details in the next question)

## What do you need help with?
  - [x] Content support
  - [x] IA support
  
I was updating the COE React widget, to use an action link to start the form, and noticed that it included an [undefined link to eBenefits to "check your eligibility"](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/coe-access/COEAccess.js#L11-L21).

- First off, I couldn't find a link to eBenefits COE status, I think it's the same as the COE form on eBenefits
- Then, I looked at the ["How to request a VA home loan" page](https://www.va.gov/housing-assistance/home-loans/how-to-request-coe/), and the action link pointing to [eBenefits COE page is essentially blank](https://www.ebenefits.va.gov/ebenefits/about/feature?feature=cert-of-eligibility-home-loan)

So my questions are about the [how to request a COE page](https://www.va.gov/housing-assistance/home-loans/how-to-request-coe/):
- What should happen to the link to the eBenefits site?
**Beth's answer:** Thanks for noticing this. We shouldn't point to eBenefits since there's no action to take there at this point. We'll add back the action link when we can point to our new online COE in the modern experience. I'll ask the sitewide content team to create a ticket to remove the action link that's currently there and to rework the content so it makes sense until we roll out our COE app.
- Once COE is ready to roll out, what content does the React widget need. It probably doesn't need a link to eBenefits. No rush on this?
**Beth's answer:** I think the content team already has a ticket for this work. I'll find it. And I think you're right that it isn't a rush.
- LGY isn't defined on this page. I've been working with COE and I still forget what it stands for...
**Beth's answer:** This isn't defined because it's the name of a tool (our Web LGY system). If LGY were standing alone in running text, we definitely would. Thank you for raising it though. I'll make a note to consider how we might reword the text. It might not be necessary to use the exact name of the tool. Will have the team assess!

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [x] Yes, but not yet
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.

See above 

## When do you expect to launch your product to 100% of users?

No set date, yet.

*Please provide an estimated date so our team can create other relevant tickets.*
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/ebenefits/apply-for-home-loan-COE
- Link to or attach user flows: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ebenefits/apply-for-home-loan-COE/coe-flow.md
- Link to prototype or mockups: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ebenefits/apply-for-home-loan-COE/coe-transition-design.md
- Link to any epics/issues that may be helpful:

## Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: Not set
- Estimated staging review date: Not set
- Content and IA work needed by: No deadline

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: Benefit team 1
-	OCTO-DE product owner: Matt Self
-	Product manager: Sade Ragsdale
-	Designer: Pamela Hastings
-	FE engineer: Robin Garrison
-	Product/team Slack channel: #benefits-team-1

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by saderagsdale** [comment]
@jsokohl review comments above and work with @Mottie as needed. Determine path forward on content that may need re-working.

**Comment by bethpottsVADEPO** [comment]
@Mottie Re: your questions above:

1. Randi will take down the entry point to nowhere until our COE form launches. (she confirmed this approach with Matt S.
2. This work is covered in another ticket.
3. We assessed and decided to leave this as is since it's the name of the tool.

**Comment by saderagsdale** [comment]
@jsokohl @Mottie  Hey both - any thoughts on next steps for this ticket?

**Comment by Mottie** [comment]
I agree with Beth's recommendations. As for the COE widget, [Jesse is working with Randi on the content](https://dsva.slack.com/archives/C5AGLBNRK/p1657650442182419).

**Comment by RLHecht** [comment]
@Mottie These changes are now live. https://www.va.gov/housing-assistance/home-loans/how-to-request-coe/ We are still working on the react widget component in another ticket.

**Comment by saderagsdale** [comment]
@Mottie Can we close this since the changes are live?

**Comment by Mottie** [comment]
This ticket is owned by the content team, it's not a ticket for our team. There is still some pending content changes for the COE React widget (action link injected into the content page), so I'd let their team determine when to close this ticket.

**Comment by RLHecht** [comment]
I'll close this ticket out. We completed the work for this and have other tickets open for when it's time to add the react widget onto the page and then for entry points once COE is launched to 100%. Is there an ETA on when the COE is going to start rolling out?

**Comment by saderagsdale** [comment]
@RLHecht Thank you! UAT for the Certificate of Eligibility is slated for the week of 8/15, which is when LGY (service provider on the backend) will be testing in parallel.

**Comment by laurwill** [comment]
Hi @Mottie @saderagsdale ,

We found 1 more pages that had the COE eBenefits link and 2 pages with a nonexistent Specially Adapted Housing eBenefits link:
- [Eligibility requirements for VA home loans](https://www.va.gov/housing-assistance/home-loans/eligibility/)
<img width="425" alt="image" src="https://user-images.githubusercontent.com/104158104/186982234-66994ffa-30bb-416c-864a-7d16d81c9956.png">

- [Disability housing grants for Veterans](https://www.va.gov/housing-assistance/disability-housing-grants/)
<img width="459" alt="image" src="https://user-images.githubusercontent.com/104158104/186982281-b3a05e47-0aa2-4fc7-afbc-4b7a3eaa964b.png">

- [How to apply for a specially adapted housing grant](https://www.va.gov/housing-assistance/disability-housing-grants/how-to-apply/)
<img width="409" alt="image" src="https://user-images.githubusercontent.com/104158104/186982326-e3c0d98b-8242-418a-970a-7daf62f8117e.png">

We'll be removing these early next week — here's the [ticket with our content work](https://github.com/department-of-veterans-affairs/va.gov-team/issues/45341) on this for your reference. Let us know if you have any concerns about this!

**Comment by saderagsdale** [comment]
@Mottie This work is done for CoE, right?

**Comment by Mottie** [comment]
@saderagsdale This ticket isn't owned by our team.

**Comment by RLHecht** [comment]
I can re-close this.
